players/default_player.py

```diff
@@ -64,7 +64,7 @@ def move(self, current_percept) -> (int, List[int]):
         if turn_number == 1:
             return constants.INIT, [0,0]
 
-        if len(polygons) != len(requests):
+        if len(polygons) < len(requests):
```

Comment:
one cut can create multiple polygons, meaning that the `!=` check might be missed. This should ensure the default player always assigns cuts to requests at some point.